spec/cli.spec.js

```diff
@@ -72,7 +72,7 @@ describe('cordova cli', () => {
         });
     });
 
-    describe('Test#004 : project commands other than plugin and platform', () => {
```

Comment:
where did this "Test#004 : project commands other than plugin and platform" go then? was this always a faulty description making no sense?

Reply:
The description "project commands other than plugin and platform", with no test number, was introduced in c775d2ab693d29b45cf5dab0583babc13685007f back in 2013. Back then the description was strictly correct but not the most helpful since the only project command tested in that section was "build". No longer correct as of 8956c75931f742a773c1e78d49c7dac85fdb6747 where tests of "config" were introduced.

Test numbering was introduced in 2652919f9d962427cd0c794335d7f907c9681dd4, where the described block starting here was mistakenly given a test number.